velox/docs/functions/spark/json.rst

```diff
@@ -11,3 +11,12 @@ JSON Functions
         SELECT json_object_keys(''); -- NULL
         SELECT json_object_keys(1); -- NULL
         SELECT json_object_keys('"hello"'); -- NULL
+
+.. function:: json_array_length(jsonString) -> bigint
```

Comment:
`.. spark:function::`

bigint -> integer

And please follow alphabetical order to add documentation.

Reply:
Done